src/EPR.Payment.Service.Common.Data/Interfaces/Repositories/Payments/IOnlinePaymentsRepository.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace EPR.Payment.Service.Common.Data.Interfaces.Repositories.Payments
+{
+    public interface IOnlinePaymentsRepository
+    {
+        Task<Guid> InsertPaymentStatusAsync(DataModels.OnlinePayment? entity, CancellationToken cancellationToken);
+        Task UpdatePaymentStatusAsync(DataModels.OnlinePayment? entity, CancellationToken cancellationToken);
+        Task<DataModels.OnlinePayment?> GetOnlinePaymentByExternalPaymentIdAsync(Guid externalPaymentId, CancellationToken cancellationToken);
```

Comment:
Does other functions names in this interface needs updating with OnlinePayment to add clarity?